charts/app-config/values-integration.yaml

```diff
@@ -2880,79 +2844,10 @@ govukApplications:
           secretKeyRef:
             name: rails-secret-key-base
             key: SECRET_KEY_BASE
-      - name: MEMCACHE_SERVERS
```

Comment:
Does whitehall-frontend not use memcache?

Reply:
Looks like it doesn't 🤷👍

```
$ sshp whitehall_frontend:1 -- govuk_setenv whitehall env |grep MEMC
...
Running command: ssh -J jumpbox.production.govuk.digital <...>.eu-west-1.compute.internal govuk_setenv whitehall env

$ 
```

Reply:
Ohh wait, there is a memcache running locally on the `whitehall_frontend` machines. Is the app defaulting to using that local memcached?

Reply:
Confirmed

```
irb(main):002:0> Rails.cache
=> 
#<ActiveSupport::Cache::MemCacheStore:0x00007f7581afe8a0                                            
 @coder=ActiveSupport::Cache::MemCacheStore::Coders::Rails70Coder,                                  
 @coder_supports_compression=true,                                                                  
 @data=                                                                                             
  #<Dalli::Client:0x00007f7581afdc20                                                                
   @key_manager=                                                                                    
    #<Dalli::KeyManager:0x00007f7581afd310                                                          
     @key_options={:digest_class=>Digest::MD5},                                                     
     @namespace=nil>,                                                                               
   @normalized_servers=["127.0.0.1:11211"],                                                         
```

Reply:
> ~Looks like it doesn't 🤷👍~
> ```
> $ sshp whitehall_frontend:1 -- govuk_setenv whitehall env |grep MEMC
> ```

Ah yeah, that's why I assumed no, but you're right should have checked for the local memcache. Will add back the env var.